s/api/types/concepts.ts

```diff
@@ -1,34 +1,43 @@
 
 export interface CommentPost {
-	id: string
-	parentCommentId: undefined | string
-	authorId: string
-	topicId: string
-	timePosted: number
-	subject: string
-	body: string
-	rating?: number
+  id: string;
```

Comment:
- indentation here is incorrect, must use tabs
- we don't use semicolons in this project, please remove them